test/common/network/socket_option_impl_test.cc

```diff
@@ -82,6 +126,12 @@ TEST_F(SocketOptionImplTest, SetOptionFreeebindSuccessFalse) {
   }
 }
 
+// Transparent settings have no effect on post-bind behavior.
```

Comment:
I thought transparent does have an effect on both prebind and postbind?

Reply:
Right. I can relax that.

Reply:
Added a commit to support setting `IP_TRANSPARENT` both pre-bind and post-bind.